### ISSUE TITLE
possibly support xcf?

### ISSUE THREAD

**Issue by somebody1234** [sindresorhus/file-type#449]
```js
  if (checkString('gimp xcf ')) {
    return ['xcf', 'image/x-xcf'];
  }
  ```
gimp's image file format
the next four bytes indicate version; however i believe this isn't needed. `file` for version 0, `v001` to `v011` for newer versions.
(of course, if this is not worth adding, feel free to close this)

**Comment by Borewit** [comment]
I am looking forward your PR @somebody1234.

**Comment by somebody1234** [comment]
closed in #450
